analysis/webservice/algorithms/imaging/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
```

Comment:
Is the binary file ```analysis/webservice/algorithms/imaging/Roboto/Roboto-Bold.ttf``` required... typically, if we can prevent it, we would be best not keeping binary in the repository.